internal/winapi/winapi.go

```diff
@@ -0,0 +1,147 @@
+package winapi
```

Comment:
So I don't know if this would be the preferred route we'd like to go but it helped me locally so far. I have the winapi directory and files split out by what API is exposed per file. So all of the constants and //sys declarations for Job objects are in jobobject.go, all of the constants and //sys declarations for tokens and logging on a user are in token.go (could be identity or logon.go but regardless) and etc etc. Makes it much easier to navigate and add things later on. Any thoughts on this? My winapi.go just has the final //go:generate go run <rest of the owl> in it.

Reply:
and yes I realize I was the one who suggested splitting by dll name originally 😄

Reply:
I think the goal for the content in winapi.go was for utilities and definitions that may be needed across different windows api functions.

Reply:
I thought the goal was to house constants and bindings for Windows APIs that are not available either in syscall or golang.org/x/sys/windows? I think some utilities are fine but that should not be the goal for this package. The goal should be low level bindings that other packages can build either higher level primitives around or just use when needed. E.g the layer management code with the file operations (winapi.NtCreateFile, winapi.FILE_OPEN etc.) and for an example from some other work with this approach https://github.com/microsoft/hcsshim/pull/861/files#diff-6a4e1f6010641d5e337001dd74783d0fR189.

Reply:
Updated. Take a look please :)